Documentation/help_files/template_maintenance_help.php

```diff
@@ -18,10 +18,12 @@
             <div class='card-title'><h5><?php echo xlt('In General'); ?></h5></div>
             <div class='card-text px-1'>
                 <ul>
-                    <li><?php echo xlt('The Scope toolbar is global to the entire form. It consists of Location, Category and action buttons. "Refresh" to clear Location selections, "Send" to send templates to patient(s), "Files" for uploading templates and a Submit to submit Location and Category current selections. By selecting location and or category, all the tables in the form will reflect those selection.'); ?></li>
+                    <li><?php echo xlt('The Scope toolbar is global to the entire view. It consists of Location, Category and action buttons.') . " <i class='fa fa-sync'></i> " .
+                     xlt('button to clear Location selections, "Send" button to send templates to patient(s) or profiles') . " <i class='fa fa-search'></i><i class='fa fa-sync'></i>" .
+                     xlt('  to search Location and Category current selections or refresh views. By selecting location and or category, all the tables in the form will reflect those selection.'); ?></li>
```

Comment:
can't have leading or ending spacing on constants (they get removed in translation pipeline and then never will translate). should put the spacing outside the constant.